tox.ini

```diff
@@ -50,12 +51,25 @@ setenv =
     links,html,pdf: BUILDER_OPTS = --color -v -j auto -W --keep-going
 commands =
     links,html,pdf: sphinx-build -d "{toxworkdir}/doc_doctree" {env:SOURCE_DIR} "{toxinidir}/{env:BUILD_DIR}/{env:BUILDER}" {env:BUILDER_OPTS} -b {env:BUILDER}
-     clean: python -c "import shutil, sys; shutil.rmtree(sys.argv[1], ignore_errors=True)" "{toxinidir}/{env:BUILD_DIR}"
+    clean: python -c "import shutil, sys; shutil.rmtree(sys.argv[1], ignore_errors=True)" "{toxinidir}/{env:BUILD_DIR}"
 
 [testenv:dist]
 description = Checks project distribution
 skip_install = true
 deps =
     build
 commands =
-    python -m build {toxinidir}
\ No newline at end of file
+    python -m build {toxinidir}
+
+[testenv:doc-preview]
```

Comment:
Let's add this with the rest of the doc environments and rename it `serve`. This is the usual name:

```ini
[testenv:doc-{clean,links,html,pdf,serve}]

skip_install:
    clean,serve: true

commands = 
    ...
    serve: stb serve "{toxinidir}/{env:BUILD_DIR}/html" --open-browser
```